thirdparty/cli-utils/preview/cmdpreview.go

```diff
@@ -169,3 +185,32 @@ func (r *PreviewRunner) RunE(cmd *cobra.Command, args []string) error {
 	printer := printers.GetPrinter(r.output, r.ioStreams)
 	return printer.Print(ch, drs)
 }
+
+// ApplyResourceGroupCRD applies the inventory ResourceGroup CRD, printing
+// output if this is the first time the CRD apply happened.
+func (r *PreviewRunner) ApplyResourceGroupCRD() {
```

Comment:
This seems like the same code as in the `ApplyRunnerWrapper`. Can we avoid the duplication?

Reply:
Fixed by moving more code into `inventoryrg.go`. The printing is still left to the commands instead of the library.